contracts/utils/VaultDeploymentHelper.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.4;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract VaultDeploymentHelper is Ownable {
+    /**
+     * @notice Emitted when a new vault is deployed
+     * @param vaultAddress the address of the new vault
+     */
+    event NewVault(address vaultAddress);
+
+    /**
```

Comment:
add constructor where we do bulk update of array for all current vaults we have

Reply:
Addressed